src/utils/supabase.js

```diff
@@ -0,0 +1,15 @@
+import { createClient } from '@supabase/supabase-js';
```

Comment:
このライブラリ、型情報含んでいるっぽいので、.tsファイルとして良いかもしれません！💡

Reply:
tsファイルに変更しました。ただ利用しないpropertyがオプショナルではないので、nullにしちゃったのですがどのように対処するのが正解でしょうか？

Reply:
```
export interface Session {
  provider_token?: string | null
  access_token: string
  /**
   * The number of seconds until the token expires (since it was issued). Returned when a login is confirmed.
   */
  expires_in?: number
  /**
   * A timestamp of when the token will expire. Returned when a login is confirmed.
   */
  expires_at?: number
  refresh_token?: string
  token_type: string
  user: User | null
}
````

Sessonの型定義みてみると、`token_type:string;`なんですね。。。
なのにtoken_typeにnullを設定していてコンパイルエラーになってないところが引っ掛かります。。。